app/models/song.rb

```diff
@@ -6,6 +6,14 @@ class Song < ApplicationRecord
 
   scope :audited, -> { joins(:audits).order('audits.time ASC') }
   scope :recently_changed, -> { where('updated_at >= ?', 1.week.ago).order(updated_at: :desc) }
+  scope :duplicates, -> {
+    where(
+      firstline_title: Song.select(:firstline_title)
+        .group(:firstline_title) # group songs into buckets of the same title
+        .having("count(*) > 1") # bucket has more than one song
+        .select(:firstline_title) # get firstline from these buckets
+    ).where("updated_at > ?", 3.months.ago) # old duplicates can be considered checked and ignored
```

Comment:
Some (dumb) songs have like entire first stanzas the same (Faith and Love album have songs like this).

Also, might want eventually to make duplicates fuzzy instead of exact, so that if someone types up the same song but the title has e.g. one character or capitalisation difference, it'll still show. If we get fuzzy, then there might be false positives in the duplicates (which will go away if ignored).

TL;DR—if for any reason there is a duplicate we want to keep, it's nice to have it not red eventually.